src/functions.js

```diff
@@ -0,0 +1,70 @@
+function capitalizeString(string) {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+}
+
+function reverseString(string) {
+  return string.split('').reverse().join('');
+}
+
+const calculator = (a, b) => {
+  const add = () => a + b;
+  const subtract = () => a - b;
+  const divide = () => a / b;
+  const multiply = () => a * b;
+
+  return {
+    add,
+    subtract,
+    divide,
+    multiply,
+  };
+};
+
+const cipher = (string, shift) => {
+  const alphabet = ['a', 'b', 'c', 'd', 'e', 'f', 'g', 'h', 'i', 'j', 'k', 'l', 'm', 'n', 'o', 'p', 'q', 'r', 's', 't', 'u', 'v', 'w', 'x', 'y', 'z'];
+
+  const phraseSplit = () => string.split('');
+  const isUppercase = letter => (letter.toUpperCase() === letter);
+  const ceasar = () => {
+    let cipherString = '';
+    phraseSplit().forEach((item) => {
+      const uppercase = isUppercase(item);
+      const letter = uppercase ? item.toLowerCase() : item;
+      const index = alphabet.indexOf(letter);
+      if (index > -1) {
+        const position = (index + shift) > alphabet.length - 1
+          ? (index + shift) - alphabet.length : (index + shift);
+
+        cipherString += uppercase ? alphabet[position].toUpperCase() : alphabet[position];
+      } else {
+        cipherString += item;
+      }
+    });
+    return cipherString;
+  };
+  return {
+    ceasar,
+  };
+};
+
+const analize = (arr) => {
+  const average = () => Math.ceil((arr.reduce((a, i) => a + i, 0)) / arr.length);
```

Comment:
I'm not sure about the ceiling of the result. The average of `[2, 3]` should be `2.5`, not `3`. Please give a second thought to this before merging.